### ISSUE TITLE
 GOD EATER RESURRECTION (460870)

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/Proton#1884]
# Compatibility Report
- Name of the game with compatibility issues: GOD EATER RESURRECTION
- Steam AppID of the game: 460870
- (God Eater 2 has the same exact problem, this is basically also a compatibility report for God Eater 2)

## System Information
- GPU: RX580
- Driver/LLVM version: Mesa 18.3.0-devel (git-7d66eddbbd) / LLVM 8-svn
- Kernel version: 4.19
- Link to full system information report: https://gist.github.com/z0z0z/544b33cb051edd5ecb16a1f47ef50e21
- Proton version: 3.16-4 beta

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

https://gist.githubusercontent.com/z0z0z/feb270e5168d2370c6181d51f74c4147/raw/02a39680e949f0915aa979b00ee93ac73ff22e7f/steam-460870.log

## Symptoms 
Game doesn't start. It gives an error dialog with a link to a codefusion url then exits. (https://support.codefusion.technology/ger/?e=88500010&l=english)

## Reproduction
Try to start game.

## Other thoughts
Game apparently used to work according to WineHQ: https://appdb.winehq.org/objectManager.php?sClass=version&iId=34449

The game does actually start in wine 3.18 staging, but crashes at initial loading screen.

Things I've tried that make no difference:

- PROTON_USE_WINED3D=1
- PROTON_NO_D3D11=1
- PROTON_NO_ESYNC=1
- Disabling all dll overrides in the prefix set by Proton
- Installing devenum/quartz through winetricks into the prefix
- Using gallium nine in wine staging outside Proton

**Comment by AyeKayKen** [comment]
how to fix the unable to initialize steam API

**Comment by AyeKayKen** [comment]
crack

**Comment by soredake** [comment]
Game works now with proton 5.0-2, slight audio cracking is present (pulseaudio + tsched=0).

[steam-460870.log](https://github.com/ValveSoftware/Proton/files/4231700/steam-460870.log)

**Comment by ghost** [comment]
The opening movie now works with the latest [Proton GE 5.8](https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/5.8-GE-1-MF) release.

**Comment by Hengecobdig** [comment]
Almost appears to work well with Proton 5.13-5, however theres an issue of extreme lag with esync off, or if I turn it on the game will somewhat randomly freeze during gameplay

**Comment by kisak-valve** [comment]
[SteamDeck] God Eater: Resurrection issue | Soft Locking

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6091.
@GingerBobRoss posted on 2022-08-14T08:40:11:

# Compatibility Report
- Name of the game with compatibility issues: God Eater Resurrection
- Steam AppID of the game: 460870

## System Information
- GPU:  GPU 0405 (Vangogh)
- Driver/LLVM version:  LLVM 12.0.0, DRM 3.45, 5.13.0-Valve21-1-Neptune-02209
- Kernel version: 5:13-Valve21-1-Neptune-02209
- Link to full system information report as https://github.com/GingerBobRoss/Gist-SystemInfo/blob/main/SteamDeckinto
- Proton version:7.03, ProtonGE 7.24, ProtonGE 7.28

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.


## Symptoms:
Game is rated as steam verified for the deck, but the game suffers from soft locks for 10-20 seconds during mission select and will soft lock mid game during hunts.
The game will sometimes have issues with audio playing as well, but the soft locking is the biggest issue to playability.


Issue can be semi resolved by using PROTON_NO_ESYNC=1 %command%, but game suffers from performance issues after playing with the command on.
Same effect when using PROTON_NO_FSYNC=1 %command% and  PROTON_NO_ESYNC=1 PROTON_NO_FSYNC=1 %command%

Additionally, The game is not available to list issues on ProtonDB, due to the game not being available on the store as it comes bundles with God Eater 2 Rage burst

**Comment by SeongGino** [comment]
Even as of today, with Proton 9 (Exp or GE), it seems like the game has to be run with either no Esync & no Fsync, *or* pinning the process to one core with `taskset -c 0` to avoid pauses when accessing new game assets or elongated load times.

Additionally, I had to be sure to install the `d3dx9_43` protontricks verb in the prefix to get the game to start and not crash when minimizing the window (which it does automatically in fullscreen when losing focus), and `wmp11` for the FMVs to work (or else showed nothing after the logos when starting, and I would just have to press a button again to continue to the title).

**Comment by wantija** [comment]
Looks like the cutscenes finally work out-of-the-box as of Proton 10's beta. Disabling esync and fsync are still required for good loading times.

**Comment by SeongGino** [comment]
If using a fork of Proton with NTsync supported (as of right now, GE-Proton10-9 is the first version with this added), loading times are fully resolved when NTsync is enabled and doesn't require core locking anymore.

Anecdotally speaking, `d3dx9_43` doesn't seem to require a manual install anymore, so the fullscreen suspending issue is fixed (at least on Wayland?).

@wantija Would you verify that the FMVs still work currently? For me, 10-1 beta, Experimental, and Hotfix all show testbars, and GE10-9 still shows a black screen and requires a manual `wmp11` install.

**Comment by wantija** [comment]
Yep the intro video still works for me with Proton experimental and Proton 10.0 beta with fresh prefixes.

**Comment by simifor** [comment]
I've had no issues with the intro video on my side, just make sure you have shader precaching enabled and it should work fine.
Proton experimental bleeding edge has changes that help loading times for this game, remove any launch parameter that affect fsync and you should notice much faster loading times. Info about how to use bleeding edge https://github.com/ValveSoftware/Proton/wiki/Proton-Versions#proton-bleeding-edge

**Comment by SeongGino** [comment]
@simifor And for the people that *don't* want shader precaching enabled...? The storage overhead is too much for my home directory which is almost constantly short on space, and I've never had to enable precaching for any of my other games.

While, yes, this *does* work (though for me requires setting to Upstream Experimental rather than GE), if this is a requirement to get preconverted FMVs, this ideally really *shouldn't* be under shader precache and should be its own thing that can be downloaded alongside the game independent of this option; since as soon as I disable precache, all precache data goes away *including the aforementioned videos* and in comes back the testbars.

**Comment by kisak-valve** [comment]
Hello @SeongGino, that request is being tracked at https://github.com/ValveSoftware/steam-for-linux/issues/9132.
